evm_loader/program/src/entrypoint.rs

```diff
@@ -296,8 +296,8 @@ fn do_finalize<'a>(program_id: &Pubkey, accounts: &'a [AccountInfo<'a>]) -> Prog
 
     if exit_reason.is_succeed() {
         info!("Succeed execution");
-        let (applies, logs) = executor.deconstruct();
-        backend.apply(applies, logs, false)?;
+        let (applies, _) = executor.deconstruct();
```

Comment:
Why you disable send events in contract constructor?

Reply:
@s-medvedev disable what? logs are not using there, they was just passing to `apply`, which also hasn't any their usage. We removed them from `apply`.